force-app/main/default/quickActions/ProgramEngagement__c.Add_to_Cohort.quickAction-meta.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<QuickAction xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>This quick action can be used to update Program Cohort on Program Engagement records in mass.</description>
```

Comment:
<description>This quick action bulk updates the Program Cohort on multiple Program Engagement records.</description>